### DISCUSSION TITLE
Why is creating a merge commit disabled?

### DISCUSSION THREAD

**Discussion by ycherniavskyi** [flutter-webrtc/flutter-webrtc#448] [General]
During reviewing and merging a pull request I found that currently "Create a merge commit" is disabled.
Is it an intentional limitation? If yes then what is the reason for it and what merge type is preferable: "Squash and merge" or "Rebase and merge"?

![image](https://user-images.githubusercontent.com/552025/104015628-9119fb80-51bd-11eb-9776-6b6d0a9bd63f.png)

**Comment by cloudwebrtc** [comment]
`Squash and merge` are usually used. The PR process often involves multiple commits, merged into a final patch and then merged, which can reduce the number of commits in the master.

**Reply by ycherniavskyi** [reply]
I see, ok.
Sorry but this one #447 I merged with "Rebase and merge". I will use "Squash and merge" in the future 😉.

**Reply by cloudwebrtc** [reply]
don't worry :)
